### PR TITLE
prezto_install rake task

### PR DESCRIPTION
**Pull request by nandalopes**
Remove zpreztorc twice symlink, avoid post-install edits on zshrc

### CODE REVIEW

**Comment by lfilho** [comment]
Hey @nandalopes I don't get what was the problem and hence what are you fixing. There's a comment in the code saying that is needed as a workaround, etc... Could you elaborate how with your code this is not needed anymore? Thanks!

File: Rakefile [hunk 1]
```diff
@@ -262,11 +262,11 @@ def install_prezto
   run %{ ln -nfs "$HOME/.yadr/zsh/prezto" "${ZDOTDIR:-$HOME}/.zprezto" }
 
   # The prezto runcoms are only going to be installed if zprezto has never been installed
-  install_files(Dir.glob('zsh/prezto/runcoms/z*'), :symlink)
-
```
Comment (nandalopes) [inline Rakefile:266]: zpreztorc is linked twice, one at line 256 and another at line 269

File: Rakefile [hunk 2]
```diff
@@ -262,12 +262,13 @@ def install_prezto
...
   # The prezto runcoms are only going to be installed if zprezto has never been installed
-  install_files(Dir.glob('zsh/zshrc'), :symlink)
   install_files(Dir.glob('zsh/prezto/runcoms/zlogin'), :symlink)
   install_files(Dir.glob('zsh/prezto/runcoms/zlogout'), :symlink)
-  install_files(Dir.glob('zsh/prezto-override/zpreztorc'), :symlink)
   install_files(Dir.glob('zsh/prezto/runcoms/zprofile'), :symlink)
   install_files(Dir.glob('zsh/prezto/runcoms/zshenv'), :symlink)
+  # Install prezto-override files
+  install_files(Dir.glob('zsh/prezto-override/zpreztorc'), :symlink)
+  install_files(Dir.glob('zsh/prezto-override/zshrc'), :symlink)
```
Comment (nandalopes) [inline Rakefile:272]: Here I moved zshrc file to the prezto-override folder, now it's not necessary to edit zshrc after symlink.

**Comment by nandalopes** [comment]
zpreztorc is linked twice on Rakefile, one at line 256 and another at line 269

I moved zshrc file to the prezto-override folder, now it's not necessary to edit zshrc after symlink.

**Comment by lfilho** [comment]
Thank you, Nanda. Could you confirm how you tested this? I remember this was very finnicky to test unless you completely uninstalled it and installed afresh. Could you confirm you tested it from scratch and it worked as expected? Thanks!

**Comment by nandalopes** [comment]
I've tested with Ubuntu Bionic and docker, first the `master` branch, and then update with `prezto-install` branch.

After I've tested directly `prezto-install` from scratch.

It's working as expected in both tests.

_PS: I've merged locally a modified Dockerfile and PR-855, cause current `docker-compose build dotfiles` stops at vim plugin installation_

**Comment by nandalopes** [comment]
I've added the modified setup for test on #858 PR

**Comment by lfilho** [comment]
Terrific. Thanks again!

**Pull request merged**